src/test/java/com/mael/payroll/paymentMethods/DepositMethodTest.java

```diff
@@ -0,0 +1,15 @@
+package com.mael.payroll.paymentMethods;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DepositMethodTest {
+
+    @Test
+    public void createsADepositMethod() {
```

Comment:
Yeah, those tests were added because the methods where created through the big FAT test that tests [*the wiring of it all*](http://octopusonalightbeam.tk/journal/2016/01/05/day059.html), and I felt insecure to have those classes there without their isolated tests.